### ISSUE TITLE
"#build 3.litematic" Chests in different directions cannot be solved

### ISSUE THREAD

**Issue by IDhammaI** [cabaletta/baritone#4455]
## Some information
Operating system:Windows 10
Java version:JDK21
Minecraft version:1.20.4
Baritone version: baritone-api-fabric-1.10.2.jar
Other mods (if used): meteor client 0.5.6  ,  sodium-fabric-0.5.8+mc1.20.4.jar litematica-fabric-1.20.4-0.17.3.jar

## Exception, error or logs
Please find your `latest.log` or `debug.log` in this folder and attach it to the issue
![image](https://github.com/user-attachments/assets/5a191db5-700e-4e26-92fe-44b9756655d8)

## How to reproduce
#build 3.litematic
![image](https://github.com/user-attachments/assets/5d708378-5abb-4cd1-a5d7-2f91c989bcb8)
the litematic like this...

## Final checklist
- [x] I know how to properly use check boxes 
- [ ] I have included the version of Minecraft I'm running, baritone's version and forge mods (if used).
- [ ] I have included logs, exceptions and / or steps to reproduce the issue.
- [ ] I have not used any OwO's or UwU's in this issue.

**Comment by ZacSharp** [comment]
Yeah, sadly directional blocks don't work very well. This is a known problem and we'd like to fix it, but, unless someone goes and contributes a pr, a fix is unlikely to happen in near future.
